api/cl_ext_image_from_buffer.asciidoc

```diff
@@ -1,19 +1,78 @@
-// Copyright 2018-2024 The Khronos Group Inc.
+// Copyright 2018-2025 The Khronos Group Inc.
 // SPDX-License-Identifier: CC-BY-4.0
 
 include::{generated}/meta/{refprefix}cl_ext_image_from_buffer.txt[]
 
 === Other Extension Metadata
 
 *Last Modified Date*::
-    2022-01-25
+    2025-01-15
 *IP Status*::
     No known IP claims.
+*Contributors*::
+  - Kevin Petit, Arm Ltd.
+  - Jeremy Kemp, Imagination Technologies
+  - Alastair Murray, Codeplay Software Ltd.
+  - Balaji Calidas, Qualcomm
 
 === Description
 
-The latest published specification for this extension is available on
-the https://registry.khronos.org/OpenCL/extensions/ext/cl_ext_image_from_buffer.html[OpenCL registry].
+This extension enables all types of images to be created from an existing buffer
+object.
+
+=== New Enums
+
+  * {cl_image_requirements_info_ext_TYPE}
+  ** {CL_IMAGE_REQUIREMENTS_SLICE_PITCH_ALIGNMENT_EXT}
+
+=== Conformance tests
+
+. Test access from kernel
+  - For all image types
+  - For a few/all image formats
+  - For several values of row/slice pitch
+  - With or without a host_ptr
+    - Create buffer and fill with data
+    - Optionally create a sub-buffer with a randomly selected offset?
+    - Create an image from the buffer
+    - Read the image from a kernel and compare with values read using the buffer and direct addressing. They must match.
+
+. TODO Test access via read/write/map commands?
+
+. TODO Test copy to/from buffer?
+
+. TODO Test fill?
+
+. TODO Test copy to/from another image?
```

Comment:
Recording a comment from the January 21st teleconference: do we want to do anything with these TODOs, or are they ok as-is?

Reply:
As discussed on the call, I'd rather make them comments in the asciidoctor source instead. Now done.